ttnn/cpp/ttnn/operations/transformer/sdpa_decode/device/sdpa_decode_op.cpp

```diff
@@ -260,7 +260,9 @@ operation::Hash ScaledDotProductAttentionDecode::compute_program_hash(
         this->is_causal,
         has_attn_mask,
         has_cur_pos,
-        input_tensors);
+        input_tensors,
+        // Hash on page_table_tensor to properly size page table CB
+        optional_input_tensors.at(1));
```

Comment:
If `has_cur_pos` or `has_attn_mask` is true, do you want to hash on those input tensors as well?

Reply:
@caixunshiren remind me, should the op miss the cache if the shape of the attention mask changes?